docs/getting_started/Install.md

```diff
@@ -0,0 +1,108 @@
+* [helm](https://helm.sh/) CLI 3 or higher**\***
+
+>**Note:** It is possible to install `oc` and `helm` inside a Python virtual environment. To do so, change to Pelorus directory (after cloning its repository), and run
+>```
+>make dev-env
+>source .venv/bin/activate
+>```
+
+## Initial Deployment
+
+To begin, clone Pelorus repository. To do so, you can run
+```
+git clone https://github.com/konveyor/pelorus.git
+```
+which will download the latest code of Pelorus.
+
+To download a stable version, run
+```
+git clone --depth 1 --branch <TAG> https://github.com/konveyor/pelorus.git
+```
+changing **TAG** by one of [Pelorus versions](https://github.com/konveyor/pelorus/tags).
+
+Change the current directory to `pelorus`, by running
+```
+cd pelorus
+```
+
+Login to the OpenShift cluster as **kube:admin** (the specific user is really necessary?), by running
+```
+oc login
+```
+
+Create the namespace called `pelorus` that Pelorus will use, by running
+```
+oc create namespace pelorus
+```
+You can choose any name you wish, but remember to change to the same name in the following commands.
+
+Pelorus gets installed via helm charts. First, deploy the operators on which Pelorus depends, by running
+```
+helm install operators charts/operators --namespace pelorus
+```
+
+Wait for the operators install to complete. (create script that checks when is finished https://github.com/hjwp/book-example/blob/master/functional_tests/base.py#L17 and I would change to an image)
+```
+$ oc get pods --namespace pelorus
+NAME                                                   READY     STATUS    RESTARTS   AGE
+grafana-operator-controller-manager-7678cc5c7c-spvls   2/2       Running   0          22s
+prometheus-operator-559d659944-fvsjg                   1/1       Running   0          10s
+```
+
+Then, deploy the core Pelorus stack, by running
+```
+helm install pelorus charts/pelorus --namespace pelorus
+```
+
+Wait for the Pelorus install to complete.
+```
+$ oc get pods --namespace pelorus
+NAME                                                   READY     STATUS      RESTARTS      AGE
+deploytime-exporter-1-deploy                           0/1       Completed   0             93s
+deploytime-exporter-1-rwk5l                            1/1       Running     0             90s
+grafana-deployment-55f77ccc8f-d7m92                    2/2       Running     0             84s
+grafana-operator-controller-manager-7678cc5c7c-spvls   2/2       Running     0             4m5s
+prometheus-operator-559d659944-fvsjg                   1/1       Running     0             3m53s
+prometheus-prometheus-pelorus-0                        3/3       Running     1 (89s ago)   93s
+prometheus-prometheus-pelorus-1                        3/3       Running     1 (89s ago)   93s
+```
+(image is outdated!! deploy just completes but does not run?)
```

Comment:
why `deploytime-exporter-1-deploy` just completes and does not run?

For me, a committime also was created
```
NAME                                                   READY   STATUS      RESTARTS      AGE
committime-exporter-1-deploy                           0/1     Completed   0             43s
committime-exporter-1-gcscq                            1/1     Running     0             40s
deploytime-exporter-1-4chlm                            1/1     Running     0             40s
deploytime-exporter-1-deploy                           0/1     Completed   0             43s
grafana-deployment-84cf947f57-xfr2k                    2/2     Running     0             34s
grafana-operator-controller-manager-86d9b6d885-6c7vb   2/2     Running     0             2m5s
prometheus-operator-7f5db89664-r7vh4                   1/1     Running     0             111s
prometheus-prometheus-pelorus-0                        3/3     Running     1 (40s ago)   44s
prometheus-prometheus-pelorus-1                        3/3     Running     1 (39s ago)   44s
```